src/mount.ts

```diff
@@ -413,16 +413,19 @@ export function mount(
   // users expect stubs to work with globally registered
   // components, too, such as <router-link> and <router-view>
   // so we register those globally.
-  // https://github.com/vuejs/vue-test-utils-next/issues/249
+  // ref: https://github.com/vuejs/vue-test-utils-next/issues/249
+  // we register both the hyphenated version and non-hyphenated
+  // version for good measure.
```

Comment:
comment is no longer accurate. suggest something like: 

```suggestion
  // we register the component as named in the stubs to avoid not being
  // able to resolve the component later due to casing
```

might also be worth adding a note in the docs about the casing being expected to match